benchmark/src/bench_client.rs

```diff
@@ -139,12 +124,82 @@ impl BenchClient {
             }
             KVClient::Etcd(ref mut etcd_client) => {
                 let response = etcd_client.get(request.key(), None).await?;
-                Ok(response.into())
+                Ok(convert::get_res(response))
             }
         }
     }
 }
 
+/// Convert utils
```

Comment:
The `From` trait might be more suitable for converting semantics.

Reply:
Orphan rules don't allow this, both types come from external dependencies